src/index.js

```diff
@@ -0,0 +1,19 @@
+import './styles.css';
+import weather from './weather';
+import location from './location';
+import display from './display';
+
+const byCity = document.getElementById('search');
+const current = document.getElementById('current');
+
+byCity.addEventListener('click', () => {
+  const locationInput = [...document.querySelectorAll('input')].map(i => i.value);
+  display(weather(location('city', locationInput)));
```

Comment:
- [ ] **[OPTONAL]** We could add a catch block to display a notification if the city is not found, instead of logging the error in the console.